### ISSUE TITLE
@sym/{ctranspose,transpose}: *transpose does not work on None

### ISSUE THREAD

**Issue by alexvong1995** [cbm755/octsympy#1215]
I run into the following issues while improving #1194.

Let `none = pycall_sympy__ ('return None')`.

Then both `none.'` and `none'` give `error: Python exception: AttributeError: 'NoneType' object has no attribute`...

I think both `none.'` and `none'` should evaluate to `none`. WDYT?

**Comment by cbm755** [comment]
I guess that's right, especially if that is convenient for other code.  I'm not really concerned about them being errors either: anyone hitting this In Real Life has likely had something go wrong...

But again, I suspect you need it for other reasons...

**Comment by alexvong1995** [comment]
Yep, we need this for our 2D-sym-compatible functions otherwise tests will fail. I think `none` should have similar semantics as `nan` so it won't be too hard to fix / test.
